### PR TITLE
chore: adding snapshot tests for nftCard components

### PR DESCRIPTION
**Pull request by JackShort**
* Adding snapshot testings for collection assets and wallet assets

### CODE REVIEW

File: src/nft/components/collection/CollectionAsset.test.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { NftStandard } from 'graphql/data/__generated__/types-and-hooks'
+import { GenieAsset, Markets, UniformAspectRatios } from 'nft/types'
+import { render } from 'test-utils/render'
+
+import { CollectionAsset } from './CollectionAsset'
+
+const AssetFixture: GenieAsset = {
```
Comment (cbachmeier) [inline src/nft/components/collection/CollectionAsset.test.tsx:7]: can you add this re-useable asset to test utils? I imagine several other nft tests will want to use a mocked not for sale asset

File: src/nft/components/profile/view/ViewMyNftsAsset.test.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { NftStandard } from 'graphql/data/__generated__/types-and-hooks'
+import { WalletAsset } from 'nft/types'
+import { render } from 'test-utils/render'
+
+import { ViewMyNftsAsset } from './ViewMyNftsAsset'
+
+const AssetFixture: WalletAsset = {
```
Comment (cbachmeier) [inline src/nft/components/profile/view/ViewMyNftsAsset.test.tsx:7]: same here about moving this to test utils

Comment (cbachmeier) [reply src/nft/components/profile/view/ViewMyNftsAsset.test.tsx:7]: Also nice way to pump your bags

File: src/nft/components/collection/CollectionAsset.test.tsx (hunk 1 above)
Comment (JackShort) [reply src/nft/components/collection/CollectionAsset.test.tsx:7]: good point

**Review by cmcewen — APPROVED** [review]
agree with @cbachmeier's comments on moving to test utils but good to merge once you do that

**Review by cbachmeier — APPROVED** [review]
LGTM

**Pull request merged**